### PR TITLE
Add descriptions for flags in the CLI

### PR DESCRIPTION
**Pull request by nol166**
### Describe in detail the problem you had and how this PR fixes it
Fills in the descriptions for flags that are missing one ( --help)

### Is there an open issue you can link to?
n/a

### CODE REVIEW

File: packages/server/src/cli.ts [hunk 1]
```diff
@@ -15,21 +15,22 @@ import { createApp } from "./server";
...
 import opn = require("opn");
+import { args } from "@oclif/parser";
 
 export class Entry extends Command {
 	public static description = "Start your own self-hosted browser-accessible VS Code";
 	public static flags = {
-		cert: flags.string(),
-		"cert-key": flags.string(),
-		"data-dir": flags.string({ char: "d" }),
+		cert: flags.string({char: "c", description: "Point to an SSL certificate .crt file"}),
```
Comment (creativeguy2013) [inline packages/server/src/cli.ts]: Should be TLS not SSL

TLS certificates don't have to end in .crt

Maybe change to "Point to a TLS certificate file"

File: packages/server/src/cli.ts [hunk 2]
```diff
@@ -15,21 +15,22 @@ import { createApp } from "./server";
...
+		"cert-key": flags.string({ char: "k", description: "Point to an SSL certificate .key file"}),
```
Comment (creativeguy2013) [inline packages/server/src/cli.ts]: Should be TLS not SSL

TLS certificate key files don't have to end in .key

Maybe change to "Point to a TLS certificate private key file"

File: packages/server/src/cli.ts [hunk 3]
```diff
@@ -15,21 +15,22 @@ import { createApp } from "./server";
...
+		"data-dir": flags.string({ char: "d", default: "Current directory", description: "Specify the working directory for the IDE"  }),
 		help: flags.help(),
 		host: flags.string({ char: "h", default: "0.0.0.0" }),
 		open: flags.boolean({ char: "o", description: "Open in browser on startup" }),
 		port: flags.integer({ char: "p", default: 8443, description: "Port to bind on" }),
 		version: flags.version({ char: "v" }),
-		"no-auth": flags.boolean({ default: false }),
-		"allow-http": flags.boolean({ default: false }),
-		password: flags.string(),
+		"no-auth": flags.boolean({ char: "n", default: false, description: "Start code-server without password" }),
```
Comment (creativeguy2013) [inline packages/server/src/cli.ts]: Change to "Start code-server without a password"

File: packages/server/src/cli.ts [hunk 4]
```diff
@@ -15,21 +15,22 @@ import { createApp } from "./server";
...
+		"allow-http": flags.boolean({ char: "a", default: false, description: "Allow use of HTTP protocol"  }),
+		password: flags.string( {description: "Specify the IDE password inline"}),
```
Comment (creativeguy2013) [inline packages/server/src/cli.ts]: Is the _inline_ necessary? Is there an other way of specifying the password?

File: packages/server/src/cli.ts [hunk 5]
```diff
@@ -234,7 +234,7 @@ export class Entry extends Command {
 			logger.warn("Launched without authentication.");
 		}
 
-		const url = `http://localhost:${flags.port}/`;
+		const url = `http${flags["allow-http"] ? "s" : ""}://localhost:${flags.port}/`;
```
Comment (kylecarbs) [inline packages/server/src/cli.ts]: I believe this is inverted.

Comment (nol166) [reply packages/server/src/cli.ts]: corrected

**Comment by kylecarbs** [comment]
I referenced these titles for descriptions in the new CLI structure. If any need to be adjusted, feel free to reopen.

**Pull request closed without merging**